packages/md-enhance/__tests__/unit/stylize.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { describe, expect, it } from "vitest";
+import MarkdownIt = require("markdown-it");
+import { stylize } from "../../src/node/markdown-it";
+import { StylizeOption } from "../../src/shared";
+
+describe("subscript", () => {
+  const options: StylizeOption = {
+    MUST: {
+      tag: ["strong", "em"],
+      attr: [
+        ["class", "badge"],
+        ["class", "tip"],
+      ],
+    },
+    SHOULD: {
+      tag: ["strong"],
+      attr: [["title", "should"]],
+    },
+    MAY: {
+      tag: ["em"],
+      text: "MAY:)",
+    },
+    NOT: {
+      tag: ["em"],
+      text: (str) => "MUST_" + str,
+    },
+  };
+  const markdownIt = MarkdownIt({ linkify: true }).use(stylize, options);
+
+  it("Should render MUST", () => {
+    expect(markdownIt.render(`**MUST**`)).toEqual(
+      '<p><strong class="badge tip">MUST</strong></p>\n'
+    );
+    expect(markdownIt.render(`*MUST*`)).toEqual(
+      '<p><em class="badge tip">MUST</em></p>\n'
+    );
```

Comment:
Other tests need to added here:

LINE TEST:

```md
**MUST** at the begining of the line.
At the end of the line **MUST**.
```

Should not break other things

```md
`**MUST**` in inline code should be rendered as is.

Other syntax like _italic_ and  **bold** should work with **MUST**.

A invalid syntax like **MUST**and_MUST_ should not be parsed.

Other word not matching keywords like **MUS** and **MUSTS** should not be parsed.
```


Please at lease complete the above tests in your test file.

Reply:
Also you should add another one: "Plugin without options should not throw"

Reply:
`**MUST**and_MUST_` can parsed by markdown-it to `<strong>MUST</strong>and_MUST_`.
so `**MUST**and` is valid syntax

Reply:
Yes, I realized that, you can change to another test string targeting this.